node/node.go

```diff
@@ -711,6 +706,11 @@ func (n *Node) OnStop() {
 			n.Logger.Error("Pprof HTTP server Shutdown", "err", err)
 		}
 	}
+	if n.indexerService != nil {
```

Comment:
Why was the ordering here changed?

Reply:
The test was failing because of it. There's an implicit assumption that DB is always closed after the indexer, which was not the case earlier.

Reply:
I'd add a comment.